src/main/scala/com/gu/ami/amiup/UI.scala

```diff
@@ -48,6 +53,27 @@ object UI extends LazyLogging {
     println("-----------------------".colour(Console.RESET))
   }
 
+  def displayRefreshProgress(refreshes: Seq[InstanceRefreshProgress]): Unit = {
+    refreshes.foreach {
+      case InstanceRefreshProgress(asgName, true, true, _, _, _) =>
+        // Instance refresh has been cancelled or has failed
+        println(s"$asgName: Instance refresh was not successful".colour(Console.RED))
```

Comment:
Is there any more info from the API calls that we can provide here? According to the types we might have a status or status reason but perhaps this won't be the case here?